### ISSUE TITLE
loss_fn by default is Mean Squared Error (MSE), not Root Mean Squared Error (RMSE)?

### ISSUE THREAD

**Issue by mw66** [KindXiaoming/pykan#32]
https://github.com/KindXiaoming/pykan/blob/915726a6770bf439021f1bac38bfe39fd424159e/kan/KAN.py#L844

Also can you add the doc for this `loss_fn` param?

**Comment by KindXiaoming** [comment]
Hi the training loss is by default MSE, but the saved results are for RMSE. RMSE loss is not continuous hence would be optimization problems.

**Comment by mw66** [comment]
> Hi the training loss is by default MSE, but the saved results are for RMSE. RMSE loss is not differentiable (at zero) hence would have optimization problems. MSE has better landscape.

OK, but can you doc this info, otherwise it's confusing when comparing the above code with the paper / tutorials.
